app/experimenter/nimbus-ui/src/components/RichText/index.test.tsx

```diff
@@ -0,0 +1,24 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from "react";
+import { render } from "@testing-library/react";
+import RichText from ".";
+
+const url = "https://www.mozilla.org/en-CA/about/manifesto/";
```

Comment:
We could also test for `http` and not just `https`. Do we want to support links provided without this like `www.mozilla.org` as well or to require it? I imagine something missing `http` / `www` can be rendered as plaintext (like `mozilla.org`), but if we want to require `http`/`https`, maybe we want to note that to the user in the form itself?

Reply:
[Updated](https://storage.googleapis.com/mozilla-storybooks-experimenter/commits/8cdd72eaf1c28a9edd50229b795414e9004f05b0/nimbus-ui/index.html?path=/story/components-richtext--basic) to support optional protocol, but note that since both `www` and `http/s` are optional, it means text with _neither_ (e.g. `foo.com`) will also get turned into a link. I think that should be fine, but it also might be a little eager and start converting things erroneously. Might need to keep an eye on it.

Reply:
I meant it seemed sufficient for `http`, `https` or `www` to render as a link and if it's missing one of those, don't render the link, but I don't mind just a `.org` getting one and maybe that mitigates needing a tooltip or some other informative text about what will be linked and what won't. Might check with @AnaMedinac / @k88hudson on this?

Reply:
Bump @AnaMedinac @k88hudson

Reply:
TLDR on above:

* Should we render all links including for URLs that just end in `.com`/`.org` etc. as links or should it be only URLs beginning with `https`/`http` and/or `www`? At the moment it's the former, [see Storybook](https://storage.googleapis.com/mozilla-storybooks-experimenter/commits/8cdd72eaf1c28a9edd50229b795414e9004f05b0/nimbus-ui/index.html?path=/story/components-richtext--basic). Is there any case where we wouldn't want a URL to link-out?
* Probably only applicable if we change the behavior from linked Storybook - do we want a note (or tooltip) somewhere to inform the user their URLs will link-out if a link has X factor (like starting with `http`/`https`)?

Reply:
Hmm, sometimes app ids are app.whatver.org so i'd probably go with just http/https/www personally, but we could see if anyone runs into it

Reply:
Updated to [match](https://regex101.com/r/3lWJNS/2) protocol and/or www, but not neither ([Storybook](https://storage.googleapis.com/mozilla-storybooks-experimenter/commits/906e6114f4e331feb3811b826578151d997c9799/nimbus-ui/index.html?path=/story/components-richtext--basic))